public_html/wp-content/themes/pattern-directory/src/components/category-context-bar/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useRef, useState } from '@wordpress/element';
+import { Flex, FlexItem } from '@wordpress/components';
+
+function CategoryContextBar( { message, title, links } ) {
+	const [ height, setHeight ] = useState( 0 );
+	const innerRef = useRef( null );
+
+	useEffect( () => {
+		if ( message ) {
+			setHeight( innerRef.current.offsetHeight );
+		} else {
+			setHeight( 0 );
+		}
+	}, [ message ] );
+
+	return (
+		<div className="category-context__bar" style={ { height: `${ height }px` } }>
+			<Flex align="center" ref={ innerRef }>
+				<FlexItem className="category-context__bar__copy">{ message }</FlexItem>
```

Comment:
This should be an `h2`, since it's functionally the archive title. And I believe this title should appear on all categories, regardless of whether it has related links.

Reply:
Yep.